charts/cloudflow-provisioning/values.yaml

```diff
@@ -0,0 +1,22 @@
+tenantId: ''
+cloudflowName: ''
+region: us-east-2
+ami: ami-073c12bdffb704056
+instanceType:
+  support: m5a.xlarge
```

Comment:
does this have to be a m5a or can it be a t3.large?

Reply:
It can be a t3 (maybe t3.xlarge, I had some issues with cattle-agent consuming a load of resources on startup and draining all the T3 instance credits). The m5a.xlarge is a couple cents more per hour over the equivalent t3.xlarge (same goes for m5a.large vs t3.large, it's a difference of 2 cents per hour without needing to worry about the overcharge for unlimited credits).